modin/core/execution/ray/generic/io/io.py

```diff
@@ -270,4 +276,4 @@ def func(df, **kw):
             lengths=None,
             enumerate_partitions=True,
         )
-        wait([part.oid for row in result for part in row])
+        ray.wait([part.oid for row in result for part in row])
```

Comment:
Here, we should use `get` function to handle errors correctly (In a separate PR)

Reply:
https://github.com/modin-project/modin/issues/3695